zkevm-circuits/src/evm_circuit/execution/push0.rs

```diff
@@ -0,0 +1,87 @@
+use crate::{
+    evm_circuit::{
+        execution::ExecutionGadget,
+        step::ExecutionState,
+        util::{
+            common_gadget::SameContextGadget,
+            constraint_builder::{ConstraintBuilder, StepStateTransition, Transition::Delta},
+            CachedRegion, Cell,
+        },
+        witness::{Block, Call, ExecStep, Transaction},
+    },
+    table::BlockContextFieldTag,
+    util::Expr,
+};
+use bus_mapping::evm::OpcodeId;
+use eth_types::Field;
+use halo2_proofs::{
+    plonk::{
+        Error,
+        Expression::Constant,
+    },
+};
+
+#[derive(Clone, Debug)]
+pub(crate) struct Push0Gadget<F> {
+    same_context: SameContextGadget<F>,
+}
+
+impl<F: Field> ExecutionGadget<F> for Push0Gadget<F> {
+    const NAME: &'static str = "PUSH0";
+
+    const EXECUTION_STATE: ExecutionState = ExecutionState::PUSH0;
+
+    fn configure(cb: &mut ConstraintBuilder<F>) -> Self {
+        // Push the value to the stack
+        cb.stack_push(Constant(F::from(0u64)));
```

Comment:
Hm, I guess this would suffice too:
```suggestion
        cb.stack_push(0.expr());
```

Seen both patterns in the codebase.